staff/carlos-calvo/e-commerce-template/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```

Comment:
You could try to write class name all in one same language, instead of mixing spanish and english